src/apps/bristleback_apps/bm_rbr/user_code/rbr_sensor.cpp

```diff
@@ -185,7 +186,11 @@ bool RbrSensor::handleDataString(const char *s, size_t read_len, BmRbrDataMsg::D
         break;
       }
       d.sensor_type = BmRbrDataMsg::SensorType::TEMPERATURE;
-      d.header.reading_time_utc_ms = rtcGetMicroSeconds(&time_and_date) / 1000;
```

Comment:
another option is to modify rtcGetMicroSeconds to return 0 if rtc is 0. That might be cleaner approach and I am open to doing that if you guys agree!

Reply:
Haha, I was thinking maybe the current return value of `rtcGetMicroSeconds` was meaningful, but Dec 9, 2178, eh? 🤣  Yes, please, modify that function to return zero if `!isRTCSet()`. Thank you!